### ISSUE TITLE
Duplicate Entry: Fatal error related to wp_redirect filter

### ISSUE THREAD

**Issue by bennemann** [GravityKit/GravityView#2176]
This is very similar to the issue reported in [#2165](https://github.com/GravityKit/GravityView/issues/2165) (which was fixed [here](https://github.com/GravityKit/GravityView/commit/ed10a0523c36430f2a99446a5b6c1f0d8f600ebb#diff-cb3397e651caaf0253c7273e6e6c4025d209a51424740e7c909a7430b6f78215R426)).

However, in this case, the redirect-related fatal error occurs when duplicating an entry instead of deleting one:
```
Fatal error: Uncaught ArgumentCountError: Too few arguments to function LiteSpeed\Control::check_redirect(), 
1 passed in /.../wp-includes/class-wp-hook.php on line 324, 
and exactly 2 expected in /.../wp-content/plugins/gravityview/includes/extensions/duplicate-entry/class-duplicate-entry.php on line 440
```

This issue was likely introduced in [this commit](https://github.com/GravityKit/GravityView/commit/9b2ba959d034143007ad8b397d8ebbb3afadf4be).

**Comment by bennemann** [comment]
https://secure.helpscout.net/conversation/2739867041/58199

**Comment by thegamerjedbest45454** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by rafaehlers** [comment]
Bennemann sent the new build to the user.

**Comment by rafaehlers** [comment]
Released in GV 2.31
